hudi-common/src/main/java/org/apache/hudi/common/table/timeline/TimelineUtils.java

```diff
@@ -343,7 +343,8 @@ public static HoodieTimeline handleHollowCommitIfNeeded(HoodieTimeline completed
     switch (handlingMode) {
       case FAIL:
         throw new HoodieException(String.format(
-            "Found hollow commit: '%s'. Adjust config `%s` accordingly if to avoid throwing this exception.",
+            "Found hollow commit: '%s'. If this happened for an incremental query, "
+                + "adjust config `%s` accordingly if prefer to avoid throwing this exception.",
```

Comment:
Lets add documentation to enum values in L360.

Reply:
I am thinking if we can change the naming of this. 
ConcurrentWritesReadStrategy
or 
OutofOrderCommitsReadStrategy.

Reply:
no doc change needed as time travel was handled in a different method